package.json

```diff
@@ -1,19 +1,50 @@
 {
 	"name": "spicetify-marketplace",
-	"version": "0.8.2",
-	"description": "Marketplace for Spicetify",
-	"license": "MIT",
-	"workspaces": [
-		"packages/*"
-	],
+	"version": "0.8.3",
+	"homepage": "https://github.com/spicetify/spicetify-marketplace",
+	"repository": {
+		"type": "git",
+		"url": "git@github.com:spicetify/spicetify-marketplace.git"
+	},
+	"bugs": {
+		"url": "https://github.com/spicetify/spicetify-marketplace/issues"
+	},
 	"scripts": {
-		"build": "yarn workspaces foreach run build",
-		"start": "yarn workspaces foreach run start",
-		"lint:marketplace": "yarn workspace @spicetify/marketplace lint"
+		"build": "spicetify-creator",
+		"build:local": "spicetify-creator --out=dist --minify",
+		"build:prod": "yarn build:local && yarn copy:docs",
+		"copy:docs": "copyfiles README.md dist/",
+		"lint": "eslint --fix src",
+		"lint:ci": "eslint src",
+		"type-check": "tsc --noEmit",
+		"watch": "spicetify-creator --watch",
+		"prepare": "husky install"
+	},
+	"engines": {
+		"node": ">=16"
 	},
-	"private": true,
-	"packageManager": "yarn@3.2.1",
```

Comment:
Oh, do we need this still?

Reply:
Those can stay I think, using a new version of Yarn is always nice